app/controllers/public.js

```diff
@@ -7,6 +7,9 @@ export default Controller.extend({
       return this.get('session.currentRouteName') !== 'public.index';
     }
   }),
+  displayEndDate: computed('model.startsAtDate', 'model.endsAtDate', function() {
+    return (this.get('model.startsAtDate') !== this.get('model.endsAtDate'));
```

Comment:
Use [`isSame`](https://momentjs.com/docs/#/query/is-same/) to do the comparison 

```suggestion
    return !this.model.startsAtDate.isSame(this.model.endsAtDate, 'minute')
```

Reply:
@niranjan94 why have you added ‘minute’ in the end of comparison?

Reply:
@kushthedude so that the comparison is done up until the minute.

in the public page we only show dates up until the minute. 
So, `2019-12-22 12:00:10` and `2019-12-22 12:00:50` should be considered as equal.